CHANGELOG.asciidoc

```diff
@@ -107,6 +107,21 @@ https://github.com/elastic/beats/compare/v6.4.0...6.4[Check the HEAD diff]
 === Beats version 6.4.0
 https://github.com/elastic/beats/compare/v6.3.1...v6.4.0[View commits]
 
+==== Known issue
+
+Due to a packaging mistake, the `modules.d` configuration directory is installed in the
+wrong path in the Metricbeat DEB and RPM packages. This manifests itself by showing an empty list
+on the `metricbeat modules list` command and failures to enable/disable any module. To workaround this issue, run the following command:
+
+[source,sh]
+-----------
+sudo cp -r /usr/share/metricbeat/modules.d /etc/metricbeat/
+-----------
+
+This should affect only new installations, upgrades should be fine because the old `modules.d` is left in place.
```

Comment:
@tsg I think this statement is not totally true, updates are still affected because they will not get the updated modules files.

Reply:
They wouldn't get the new configs, but @ruflin checked and he said there weren't significant changes. I was trying to say that the worse aspect of this (Metricbeat doing nothing when started) will only happen on new installations. Not sure if there's a better wording that's also succinct enough.

Reply:
<3 @dedemorton